README.md

```diff
@@ -26,14 +29,17 @@ Options:
   -d, --database            SQLite database file or :memory: [string] [required]
   -p, --port                Port to listen on                           [number]
   -r, --rate-limit-enabled  Enable rate limiting                       [boolean]
+  -c, --cors                CORS white list URL values                 [string]
```

Comment:
A better description for this option is `CORS whitelist origins`.

Reply:
Hello @thevahidal,

Thank you for reviewing the PR. I really appreciate it. I have made modifications based on your comments. Please let me know if there is anything else that needs to be fixed.